src/components/pages/lessons/LessonInfo.tsx

```diff
@@ -156,6 +156,8 @@ const LessonInfo: FunctionComponent<LessonInfoProps> = ({
   lesson,
   ...restProps
 }) => {
+  const code_url = get(lesson, 'code_url')
```

Comment:
you could eliminate these lines by using `if(lesson?.code_url)` below

Reply:
oh, not completely

Reply:
pushed, does it what you mean?